utils/manager.py

```diff
@@ -63,3 +63,53 @@ async def delete(self, word: str):
             await cur.execute(f'DELETE FROM wordfilter WHERE word="{word}"')
         await self.load()
         return word
+
+
+class Invite:
+    def __init__(self, invite_code, remaining_uses):
+        self.code = invite_code
+        self.uses = remaining_uses
+
+    @property
+    def is_temporary(self):
+        return self.uses > 0
+
+
+class InviteFilterManager(Manager):
+    def __init__(self, bot):
+        super().__init__(bot)
+        self.invites = {}
+
+    async def load(self):
+        self.invites.clear()
+        invites = await self.fetch()
+        if invites:
+            for code, _, alias, uses in invites:
+                self.invites[alias] = Invite(code, uses)
+        print("Loaded Invites")
+
+    async def add(self, code: str, name: str, alias: str, uses: int):
+        async with self.dbcon as cur:
+            try:
+                await cur.execute(f"INSERT INTO invitefilter VALUES ('{code}', '{name}', '{alias}', {uses})")
```

Comment:
Hax suggested this instead: `await cur.execute("INSERT INTO invitefilter VALUES (?, ?, ?, ?)", (code, name, alias, uses))` (it prevents SQL injection and also seems to fix an issue with adding certain servers to the list..).